spec/lib/smart_name_spec.rb

```diff
@@ -2,8 +2,16 @@
 require File.expand_path('../spec_helper', File.dirname(__FILE__))
 require 'core_ext'
 
-
 describe SmartName do
+  before :all do
```

Comment:
Probably would be better to add this to a new class inheriting from SmartName, but at least there are a couple of tests to start with.